src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt

```diff
@@ -127,11 +129,34 @@ class DataCenterFormula private constructor(
     ): ProvisionedJira = time("provision Jira Data Center") {
         logger.info("Setting up Jira...")
 
-        val executor = Executors.newCachedThreadPool(
-            ThreadFactoryBuilder()
-                .setNameFormat("data-center-provisioning-thread-%d")
-                .build()
-        )
+        AbruptExecutorService(
```

Comment:
This is new in `concurrency:1.2.0`, we need to declare that `implementation` dependency explicitly.

- [ ] `implementation("...:concurrency:[1.2.0, 2.0.0)"`